### ISSUE TITLE
Backspace deletes a whole document on Firefox and Safari

### ISSUE THREAD

**Issue by dzafel** [tinymce/tinymce#3459]
Sometimes when selecting just one sentence or few paragraphs and hitting backspace key the whole document is removed. I could see that bug on Firefox (50.1) and Safari (10.0.2) on macOS Sierra. I don't have a Windows PC now to test. Chrome on mac seems to be working good. Please see this video: https://youtu.be/yI3lN_icpPQ

Sample code used to test a bug:
```
<p>Test test</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>test1</p>
<p>test2</p>
```

**Comment by fyrkant** [comment]
Fixed in yesterdays 4.5.3 release.

**Comment by pavel-j** [comment]
The same bug seems to be even in TinyMCE 4.5.3 - in Google Chrome, Windows 10.

I select text, press Del and entire document is blanked. Unfortunately, this happens just sometimes
(about three times a day / tinymce used in e-mail app).

Additional detail - after the document is blanked (by this bug), it is possible to undo (undo also restores the same selection that was before). When I hit Del again then, it is working OK - no document deleted.
